### PR TITLE
Document auth settings for #146

### PR DESCRIPTION
**Pull request by olenagerasimova**
For #146 I've added description about auth configuration in readme.

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#173](https://github.com/artipie/artipie/issues/173) is assigned to @paulodamaso[/z](https://www.0crat.com/u/paulodamaso), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/36bad358-c05f-4f13-b431-5f4804ee3ffa); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: README.md [hunk 1]
```diff
@@ -426,9 +411,36 @@ NPM js proxy will cache all artifacts from npmjs, so if somebody decides to remo
...
 ```
 
-### Credentials settings
+## Authentication
+
+For now, we have two ways to configure authentication in Artipie: via environment variables and via 
+yaml credentials file. Authentication type should be set in main Artipie config inside `meta`
+section:
+```yaml
+credentials:
```
Comment (g4s8) [inline README.md:420]: @olenagerasimova I'd mention that it should be under `meta` element to avoid confusion:
```yaml
meta:
  credentials:
    ...
```

**Comment by 0crat** [comment]
This pull request [#173](https://github.com/artipie/artipie/issues/173) is assigned to @g4s8[/z](https://www.0crat.com/u/g4s8), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/e12103c9-928b-4cd9-8d3a-1b059ec2b236); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be no monetary reward for this job

**Comment by olenagerasimova** [comment]
@g4s8 thanks, corrected

**Comment by olenagerasimova** [comment]
@g4s8 ping

**Comment by 0crat** [comment]
Job was finished in 24 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
Job audit: `ARC` didn't merge or close PR, performer @g4s8[/z](https://www.0crat.com/u/g4s8)[/z](https://www.0crat.com/u/g4s8) didn't make CR comments

**Pull request merged**